js/layout-javascript/agenda-code.js

```diff
@@ -2403,7 +2411,37 @@ DynamicList.prototype.addDetailViewData = function(entry) {
       content = entry.originalData[dynamicDataObj.column];
     }
 
-    content = _this.Utils.String.toFormattedString(content);
+    if (dynamicDataObj.type === 'image') {
+      content = entry.originalData[dynamicDataObj.column];
+
+      var contentArray;
+
+      if (typeof content === 'string') {
+        var detectURLRegex = /((?:ftp|http|https):\/\/(?:\w+:{0,1}\w*@)?(?:\S+)(?::[0-9]+)?(?:\/|\/(?:[\w#!:.?+=&%@!-/]))?)/;
```

Comment:
Please add a comment to explain what this RegExp is meant to do.

I also see this process repeated several times in all the layout JS files. Can we consolidate them to keep it DRY?

Reply:
@YaroslavOvdii I don't see this being responded to or dealt with. I still see this being repeated a lot and wonder if it can be modularised through `utils.js`.